aicsimageio/aics_image.py

```diff
@@ -625,6 +696,36 @@ def physical_pixel_sizes(self) -> PhysicalPixelSizes:
         """
         return self.reader.physical_pixel_sizes
 
+    def get_mosaic_tile_position(self, M: int) -> Optional[Tuple[int, int]]:
```

Comment:
can we lowercase the m variable?  or give it a verbose name like mosaic_tile_index

Reply:
I will make a fully qualified name: `mosaic_tile_index`